custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -81,7 +82,10 @@ def _state_aggregation_query(self, state_id):
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
             start_month_string
         )
-        seeking_services = "(person_cases.registered_status IS DISTINCT FROM 0 AND person_cases.migration_status IS DISTINCT FROM 1)"
+        migration_status = "agg_migration.is_migrated=1 AND agg_migration.migration_date < {}".format(
+            start_month_string)
+        seeking_services = "(person_cases.registered_status IS DISTINCT FROM 0 AND {} IS DISTINCT FROM 1)".format(
```

Comment:
here migration_status will be a bool no?

Reply:
nope, using a small int instead